spec/services/grant_code_exchanger_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe GrantCodeExchanger do
+  describe '#run' do
+    it 'saves the access_token and refresh_token for the sandwich' do
+      refresh_token_from_fixture = 'fake-refresh-token'
+      access_token_from_fixture = 'fake-access-token'
+      heroku_uuid = 'some-uuid'
+      Sandwich.create!(heroku_uuid: heroku_uuid)
+
+      GrantCodeExchanger.new(heroku_uuid: heroku_uuid, oauth_grant_code: 'some-code').run
+      sandwich = Sandwich.find_by(heroku_uuid: heroku_uuid)
+
+      expect(sandwich.access_token).to eq access_token_from_fixture
+      expect(sandwich.refresh_token).to eq refresh_token_from_fixture
+    end
+
+    it 'saves the datetime when the access token expires' do
+      Timecop.freeze do
```

Comment:
There's the built-in [time helpers](http://api.rubyonrails.org/v5.1/classes/ActiveSupport/Testing/TimeHelpers.html), which might use Timecop internally, I dunno.

Reply:
oh cool, never knew about that! Do you think it would be better to use those rather than timecop? I am very used to timecop but this looks cool